dbscan/example_test.go

```diff
@@ -6,7 +6,7 @@ import (
 
 func ExampleScanAll() {
 	type User struct {
-		ID    string `db:"user_id"`
+		ID    string `db:"user_id,other_tag_value"`
```

Comment:
I don't think we should add this to examples. It might be rather confusing because the majority of users won't have any `other_tag_value` to pass to this library.
We should add a note about supporting commas in the struct tag to the docs though. I would add to the bottom of this doc section: https://pkg.go.dev/github.com/georgysavva/scany@v0.2.8/dbscan#hdr-Mapping_struct_field_to_database_column

Something like that:

> dbscan supports commas "," in the struct tag name. That makes it compatible with the struct tag formats of other libraries. dbscan splits the tag name by "," and uses the first part as the column name. So `db:"user_id,other_tag_value"` struct tag is equivalent to `db:"user_id"` for dbscan.

Reply:
sure I was just looking for an easy place to add it to the unit tests.  should I just add another _test file?